openc3/lib/openc3/conversions/conversion.rb

```diff
@@ -67,6 +70,7 @@ def as_json(*_a)
       result['converted_type'] = @converted_type if @converted_type
       result['converted_bit_size'] = @converted_bit_size if @converted_bit_size
       result['converted_array_size'] = @converted_array_size if @converted_array_size
+      result['args'] = JSON.generate(@args) if @args
```

Comment:
as_json shouldn't call JSON.generate.  The goal is return objects that are ready to be converted to JSON, but not return an actual JSON string yet.   Change to "= @params.as_json(*a)" and remove underscore from the existing _a